microsoft-edge/accessibility/build/index.md

```diff
@@ -124,9 +124,10 @@ SSB BART Group is a digital accessibility firm supporting their clients in devel
 
 JavaScript library to help modern web applications with accessibility concerns by making accessibility simpler.
 
-#### [Heydonworks - ARIA Examples](http://heydonworks.com/practical_aria_examples/)
+<!--  website no longer exists  -->  
+<!--  #### [Heydonworks - ARIA Examples](http://heydonworks.com/practical_aria_examples/)
```

Comment:
I would just cut this section entirely. I couldn't find examples but we can also just point to: https://developer.mozilla.org/docs/Web/Accessibility/ARIA